plugins/DynamicsProcessor/DynamicsProcessor.cpp

```diff
@@ -153,14 +155,17 @@ bool DynProcEffect::processAudioBuffer( sampleFrame * _buf,
 		for ( i=0; i <= 1; i++ )
 		{
 			const double t = m_rms[i]->update( s[i] );
+			// smoothing "t" to filter unwanted noise
+			const double dif = qMin(std::abs(t-m_smoothRms[i])*0.1, 0.05);
```

Comment:
I would prefer we use `std::min` .

```suggestion
			const double dif = std::min(std::abs(t-m_smoothRms[i]) * 0.1, 0.05);
```